pkgs/development/ocaml-modules/hmap/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildOcaml
+, fetchurl
+, topkg
+}:
+
+buildOcaml rec {
```

Comment:
Please don’t use `buildOcaml`.

Reply:
Is there a deprecation notice somewhere?

Reply:
It is not documented.